docs/book/how-to/customize-docker-builds/use-a-prebuilt-image.md

```diff
@@ -0,0 +1,114 @@
+# Skip building an image on every pipeline run
```

Comment:
This already gets done as code is uploaded to artifact store

Reply:
you're right. what could be a better title? how about "Use a pre-built image for pipeline execution"